OrbitCore/SymbolsClient.cpp

```diff
@@ -30,23 +30,24 @@ void SymbolsClient::LoadSymbolsFromModules(
   for (const std::shared_ptr<Module>& module : modules) {
     if (module == nullptr) continue;
     ModuleDebugInfo module_info;
-    const char* name = module->m_Name.c_str();
-    module_info.m_Name = name;
+    const char* module_name = module->m_Name.c_str();
+    module_info.m_Name = module_name;
```

Comment:
I do not think this is needed.. you can probably just assign module->m_Name here, and use it below for logging as well.